src/components/ActualTodo/ActualTodo.tsx

```diff
@@ -0,0 +1,109 @@
+import classNames from 'classnames';
+import React, { useContext, useState } from 'react';
+import { Todo } from '../../types/Todo';
+import { ProcessedContext } from '../ProcessedContext/ProcessedContext';
+import { UpdateTodoForm } from '../UpdateTodoForm/UpdateTodoForm';
+
+interface Props {
+  todo: Todo,
+  onDelete: (todoId: number) => void,
+  onUpdate: (todoId: number, dataToUpdate: Partial<Todo>,) => void,
+}
+
+export const ActualTodo: React.FC<Props> = ({
```

Comment:
it will be more understandable to name it TodoItem  (just make it start from "Todo" word))